src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,130 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    //
+    private static final Manufacturer FIRST_MANUFACTURER =
+            manufacturerService.create(new Manufacturer("Audi","Germany"));
+    private static final Manufacturer SECOND_MANUFACTURER =
+            manufacturerService.create(new Manufacturer("Ford", "America"));
+    private static final Manufacturer THIRD_MANUFACTURER =
+            manufacturerService.create(new Manufacturer("Opel", "Germany"));
+    //
+    private static final Driver FIRST_DRIVER =
+            driverService.create(new Driver("Frank", "12345"));
+    private static final Driver SECOND_DRIVER =
+            driverService.create(new Driver("Alex", "67890"));
+    private static final Driver THIRD_DRIVER =
+            driverService.create(new Driver("Bob", "54321"));
+    //
+    private static final List<Driver> FIRST_CAR_DRIVERS =
+            new ArrayList<>(List.of(FIRST_DRIVER));
+    private static final List<Driver> SECOND_CAR_DRIVERS =
+            new ArrayList<>(List.of(SECOND_DRIVER, THIRD_DRIVER));
+    private static final List<Driver> THIRD_CAR_DRIVERS =
+            new ArrayList<>(List.of(FIRST_DRIVER));
+    private static final List<Driver> FOURTH_CAR_DRIVERS =
+            new ArrayList<>(List.of(FIRST_DRIVER));
```

Comment:
I dont think we need this fields as constant fields, lets make them local variables